src/style.css

```diff
@@ -0,0 +1,72 @@
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+  padding: 0 50px;
+}
+
+.logo-box {
+  width: 40px;
+  height: 40px;
+}
+
+.logo-box img {
+  width: 100%;
+  height: 100%;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+.nav-item {
+  margin-right: 20px;
+}
+
+.nav-item:nth-last-of-type(1) {
+  margin-right: 0;
+}
+
+.nav-link {
+  font-family: "Roboto";
```

Comment:
You can write it without brackets.
Which family will be displaying if google fonts not responding?
You should add additional families like `Arial` or `sens-serif`.